crates/biome_js_analyze/src/lint/nursery/use_sorted_classes.rs

```diff
@@ -161,7 +162,19 @@ impl Rule for UseSortedClasses {
 
         if node.should_visit(options)? {
             if let Some(value) = node.value() {
-                let sorted_value = sort_class_name(&value, &SORT_CONFIG);
+                // in some case we need to ignore the last element
+                // for example, in the case of template literals <div class={`mx-2 m-5 bar-${variable}`} /> we should ignore the last element
```

Comment:
The logic looks good, but the whole naming and comments orbit around the concept of "last class", but it's not correct because:
- you added test cases where the class to ignore is in the middle and at the beginning;
- you use `next_sibling`, which isn't the "last class"